.github/workflows/nightly.yml

```
@@ -14,13 +14,13 @@ jobs:
       uses: actions/checkout@master
       with:
         submodules: 'recursive'
-    - task: Npm@1
-      inputs:
-        command: 'install'
-        workingDir: 'Apps'
-      displayName: 'Install Babylon.js NPM packages'
     - name: Make Solution
-      run: mkdir buildWin32_x64 && 
+      run: mkdir Apps\node_modules &&
+        mkdir Apps\node_modules\babylonjs &&
+        mkdir Apps\node_modules\babylonjs-gui &&
+        mkdir Apps\node_modules\babylonjs-loaders &&
+        mkdir Apps\node_modules\babylonjs-materials &&
+        mkdir buildWin32_x64 && 
         cd buildWin32_x64 &&
```

`&&` isn't necessary

```suggestion
      run: mkdir Apps\node_modules
        mkdir Apps\node_modules\babylonjs
        mkdir Apps\node_modules\babylonjs-gui
        mkdir Apps\node_modules\babylonjs-loaders
        mkdir Apps\node_modules\babylonjs-materials
        mkdir buildWin32_x64
        cd buildWin32_x64
```

---

Might also consider separating this into individual steps before calling curl. It's kind of weird to have these commands in the Make solution step.

---

i was able to test this out in my fork, we needed to split this up so that we have the curl downloaded files prior to cmake generation too